crates/matrix-sdk/src/sliding_sync/list/mod.rs

```diff
@@ -260,6 +260,12 @@ impl SlidingSyncList {
     pub fn sync_mode(&self) -> SlidingSyncMode {
         self.inner.sync_mode.read().unwrap().clone()
     }
+
+    /// Sets some rooms in the filled state.
+    pub fn set_filled_rooms(&self, rooms: Vec<OwnedRoomId>) {
```

Comment:
Do we want this to be part of the public API? I don't believe so. Can we switch to `pub(crate)`?

Reply:
This is defined in a `#[cfg(testing)]` block, so I was supposing that's fine?

Reply:
Yeah! Of course! Sorry for the noise.